### DISCUSSION TITLE
Invalidate Identity?

### DISCUSSION THREAD

**Discussion by bombillazo** [refinedev/refine#4074] [General]
Hello, how can I invalidate the user identity so it is re-fetched if some request was sent to the backend responded successfully? I wish to refresh the UI to reflect any changes in the user data.

**Comment by alicanerdurmaz** [comment]
Hello, we have a `useInvalidateAuthStore` hook that invalidates the auth store, but it's currently not exported. I will discuss with the core team to export it, but in the meantime, you can copy-paste the code and use it to invalidate the auth store from anywhere in your code.

By the way, you need to install `"@tanstack/react-query"`.
